fendermint/vm/message/src/ipc.rs

```diff
@@ -128,6 +128,8 @@ pub struct FinalizedBlob {
     pub subscriber: Address,
     /// The blake3 hash of the blob.
     pub hash: Hash,
+    /// The size of the blob.
+    pub size: u64,
```

Comment:
`FinalizedBlob` needs to have size to construct `BlobPoolItem` again so it can be removed from the pool.